update_service/index.adoc

```diff
@@ -0,0 +1,48 @@
+:context: update-service
+[id="update-service"]
+= Understanding the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+[id="update-service-install"]
+== Installing the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
+// You can install the {product-title} Update Service by deploying the Update Service operator, using the {product-title} xref FIXME ../../update_service/install/web-console.adoc[web console] or xref FIXME ../../update_service/install/cli.adoc[CLI].
```

Comment:
```suggestion
// You can install the {product-title} Update Service by deploying the Update Service Operator by using the {product-title} xref FIXME ../../update_service/install/web-console.adoc[web console] or xref FIXME ../../update_service/install/cli.adoc[CLI].
```
I'd probably remove the xrefs and say something about "by using one of the following sections." If you keep the xrefs, you'll need to include an explicit anchor to the first heading using the H1 anchor name from the module and the context statement from the assembly. (assuming that you are trying to link to the next two modules.)

Reply:
Did the `,` -> `by` with adf5ca0732 -> 2c77ff9ce2

I will try to sort out the xrefs in a future reroll; I'd like to make it easy for folks to skip forward to view the approach they are interested in with a click here, but I may end up throwing in that towel and using your `by using one of the following sections.`.